singolo/sass/footer/_footer.scss

```diff
@@ -0,0 +1,71 @@
+.footer{
+    background-color: #2D303A;
+}
+.footer-content{
+    width: 1020px;
```

Comment:
You have several places with the same styles:

- width: 1020px;
- margin: 0px auto;
- What abou common class?

Reply:
fixed